source/documentation/runbooks/010-disaster-recovery-plan.html.md.erb

```diff
@@ -0,0 +1,129 @@
+| Disruption to hosting services                                                                                     | High     | Low        | - Following recovery of the platform, we may need to [re-deploy our applications](#issues-at-cloud-platform-level) |
+| AWS Region outage                                                                                                  | High     | Low        | - Wait<br>- This may lead to Disruption to hosting services (above) |
+| Loss of data due to misconfiguration of AWS resources in cloud-platform-environments repo                          | Medium   | Low        | - [Restore database from RDS snapshot](#database-recovery) |
+| Intentional disruption by malicious actor e.g. Distributed Denial of Service (DDoS)                                | High     | Medium     | - [Horizontal auto-scaling] with sensible scaling ceiling<br>- Rate limiting on NGINX ingress<br>- [AWS Shield Standard] (managed DDoS protection service) safeguards applications running on AWS. CP are looking into rolling out [AWS Shield Advanced] for more comprehensive integrated protection |
+| Deletion / corruption of application database data                                                                 | Medium   | Low        | - [Restore from snapshot](#database-recovery) |
+| Dependency poisoning attacks                                                                                       | High     | Low        | - Using Dependabot<br>- Scanning on pipeline - NPM audit<br>- Container scanning<br>Dependency assessment criteria<br>Guardduty monitors VPC flow logs for unusual activity |
+| Accidental misconfiguration of code leading to data loss                                                           | High     | Low        | - Manually fix in the DB where possible<br>- DB snapshot	restore |
+| Entra ID disruption (loss of authentication function)                                                              | High     | Low        | - Wait |
+
+## Backup Procedures
+
+### Django application
+- **Application code**:
+   - All application code is pushed to the [public GitHub repository].
+
+### Databases
+- **PostgreSQL database in AWS RDS (Find MoJ data, DataHub)**:
+    - RDS automated daily snapshots are enabled by default in CP.
+    - Ensure point-in-time recovery is active.
+    - `TODO` Schedule regular manual snapshots and store them in a different AWS region.
+
+    >**Note** manual snapshots set up this way must also be deleted, as there is an account-level snapshot quota.
+- **OpenSearch (DataHub)**:
+    - OpenSearch index snapshots are [handled by Cloud Platform as part of OpenSearch deployment]
+
+## Recovery Procedures
+
+### Django application
+<placeholder>
+
+### Microsoft Entra ID integration
+<placeholder>
```

Comment:
The Azure EntraID tenant is not managed by this team, and any significant outage would cause much wider disruption than just to the Data Catalogue. The only course of action in this event would be to monitor communications on the issue.

The more likely scenario - which is in scope for this team - is that the API keys/secrets which the applications use to communicate with the EntraID tenant expire or are deleted. In this event, follow the {instructions here}